src/com/linkedin/parseq/ParSeqGlobalConfiguration.java

```diff
@@ -58,4 +60,30 @@ public static boolean isCrossThreadStackTracesEnabled() {
   public static void setCrossThreadStackTracesEnabled(boolean enabled) {
     _crossThreadStackTracesEnabled = enabled;
   }
+
+  /**
+   * Returns true if trampoline is currently enabled.
+   *
+   * Trampoline prevents stack overflow in situation of extremely deep large ParSeq plans. Typically this problem
+   * does not occur and trampoline can be disabled allowing performance optimization.
```

Comment:
This flag is used by Continuations class which I think is decently documented. Do you have questions about Continuations class?